src/relay/backend/compile_engine.cc

```diff
@@ -612,11 +613,20 @@ class MakeShapeFunc : public backend::MemoizedExprTranslator<Array<te::Tensor>>
 class CompileEngineImpl : public CompileEngineNode {
  public:
   // Lower the function.
-  CachedFunc Lower(const CCacheKey& key) { return LowerInternal(key)->cached_func; }
+  CachedFunc Lower(const CCacheKey& key) {
```

Comment:
I think we can have just one Lower function -- is there a reason why mangling the name is bad ?

cc : @areusch